consts.php

```diff
@@ -107,12 +107,131 @@
     ],
 ];
 
+// use hardcoded.php to bulk update update this after creating a .cow.pat.json
+// for multiple versions, use an array e.g. silverstripe-mymodule => ['2.3', '2.4']
 const INSTALLER_TO_REPO_MINOR_VERSIONS = [
     '4.10' => [
-        'silverstripe-html5' => '2.3',
+        'agency-extensions' => '2.6',
+        'comment-notifications' => '2.2',
+        'cwp' => '2.9',
+        'cwp-core' => '2.9',
+        'cwp-pdfexport' => '1.3',
+        'cwp-search' => '1.6',
+        'silverstripe-advancedworkflow' => '5.6',
+        'silverstripe-akismet' => '4.2',
+        'silverstripe-auditor' => '2.4',
+        'silverstripe-blog' => '3.9',
+        'silverstripe-ckan-registry' => '1.4',
+        'silverstripe-comments' => '3.7',
+        'silverstripe-composer-update-checker' => '2.1',
+        'silverstripe-config' => '1.3',
+        'silverstripe-content-widget' => '2.3',
+        'silverstripe-contentreview' => '4.4',
+        'silverstripe-crontask' => '2.4',
+        'silverstripe-documentconverter' => '2.2',
+        'silverstripe-elemental' => '4.8',
         'silverstripe-elemental-bannerblock' => '2.4',
+        'silverstripe-elemental-fileblock' => '2.3',
+        'silverstripe-elemental-userforms' => '3.1',
+        'silverstripe-environmentcheck' => '2.4',
+        'silverstripe-externallinks' => '2.2',
+        'silverstripe-fluent' => '4.6',
+        'silverstripe-fulltextsearch' => '3.9',
+        'silverstripe-graphql' => '3.7',
+        'silverstripe-gridfieldqueuedexport' => '2.6',
+        'silverstripe-html5' => ['2.2', '2.3'],
```

Comment:
Does the matrix gen know how to handle arrays like this for the module branch?

Reply:
updated